2022-07_sprint/sprint_parties/contributors.md

```diff
@@ -447,7 +447,7 @@ Daniel Saunders
 :::
 :::{grid-item}
 
-<img alt="Image for contributor template" src="../../_static/contributors/blank_person.jpg" class="rounded-circle">
+<img alt="Image for contributor template" src="../../_static/contributors_2022_07/daniel_saunders.jpg" class="rounded-circle">
```

Comment:
As you can see in the preview: https://pymc-data-umbrella--145.org.readthedocs.build/en/145/2022-07_sprint/sprint_parties/contributors.html if you scroll down, you are modifying the image for Yuan Li, not your own.

All the info for each contributor: image, name, location, social links; is inside the same `grid-item`. That is, below a `:::{grid-item}` opening and a `:::` closing, each `grid-item` enclosing corresponds to a different contributor

Reply:
oops! I see my mistake now. Thanks for explaining that.